src/components/shared/Alert/index.scss

```diff
@@ -0,0 +1,16 @@
+.usa-alert__body {
+    padding-right: 2rem !important;
+    max-width: 100% !important;
```

Comment:
This line cancels out the max-width set on the [TRB homepage info alert](https://github.com/CMSgov/easi-app/blob/1169d804cc7efde37d9efcd540bb247b0d1f4c4c/src/views/TechnicalAssistance/index.scss#L25C5-L27):

![Screenshot 2023-05-18 at 2 15 22 PM](https://github.com/CMSgov/easi-app/assets/60187543/ea3f198f-1846-4bf8-8762-f7bef1dfa796)

Is `!important` necessary here or could that be removed? `max-width: none` is set on the alert body in [custom.scss](https://github.com/CMSgov/easi-app/blob/1169d804cc7efde37d9efcd540bb247b0d1f4c4c/src/stylesheets/custom.scss#L115) so not sure if that essentially achieves the same thing.